ui/assets/icons/Workspace.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+const WorkspaceIcon = ({ width, height, fill, style = {}, secondaryFill }) => (
```

Comment:
Adding default prop value will be helpful
```suggestion
const WorkspaceIcon = ({ width = 24, height = 24, fill, style = {}, secondaryFill }) => (
```

Reply:
I'm going to use icons from sistent. So those icons are temporary added. I'll replace them.